yt_dlp/extractor/rai.py

```diff
@@ -350,22 +455,6 @@ class RaiIE(RaiBaseIE):
         'params': {
             'skip_download': True,
         },
-    }, {
-        # ContentItem in iframe (see #12652) and subtitle at 'subtitlesUrl' key
-        'url': 'http://www.presadiretta.rai.it/dl/portali/site/puntata/ContentItem-3ed19d13-26c2-46ff-a551-b10828262f1b.html',
-        'info_dict': {
-            'id': '1ad6dc64-444a-42a4-9bea-e5419ad2f5fd',
-            'ext': 'mp4',
-            'title': 'Partiti acchiappavoti - Presa diretta del 13/09/2015',
-            'description': 'md5:d291b03407ec505f95f27970c0b025f4',
-            'upload_date': '20150913',
-            'subtitles': {
-                'it': 'count:2',
-            },
-        },
-        'params': {
-            'skip_download': True,
-        },
```

Comment:
> ContentItem in iframe (see #12652) and subtitle at 'subtitlesUrl' key

Why was this test removed? If it is obsolete, can we add equivalent test?

Reply:
the website doesn't exist anymore.

Rai restructured it's website completely removing tv-shows subdomains for each show ( TVSHOW.rai.it ) and now all the tv-shows are in the raiplay.it website  ( raiplay.it/type-of-show/TVSHOW )